packages/block-editor/src/hooks/letter-spacing.js

```diff
@@ -45,6 +45,7 @@ export function LetterSpacingEdit( props ) {
 		<LetterSpacingControl
 			value={ style?.typography?.letterSpacing }
 			onChange={ onChange }
+			__unstableInputWidth={ false }
```

Comment:
Similarly, I'm not sure if boolean value is correct here?

Reply:
I've [switched this to `null`](https://github.com/WordPress/gutenberg/pull/36367/files#diff-7f6873e89e742b9f974a2b72f19ac8dc16b704d513aa0c2d31e29be331519c95R48) instead to override the default width required for backwards compatibility purposes.